h2/src/test/org/h2/test/unit/TestPgServer.java

```diff
@@ -270,6 +270,14 @@ private void testPgClient() throws SQLException {
         rs.next();
         assertEquals(1, rs.getInt(1));
 
+        rs = stat.executeQuery("select array_to_string(array[null, 0, 1, null, 2], ',')");
+        rs.next();
+        assertEquals("0,1,2", rs.getString(1));
+
+        rs = stat.executeQuery("select array_to_string(array['a', null, '', 'b', null], ',')");
+        rs.next();
+        assertEquals("a,,b", rs.getString(1));
+
```

Comment:
We prefer pure SQL tests for such functions, see `TestScript` and `.sql` files.

Reply:
moved to `compatibility.sql`